src/pages/resultsView/plots/PlotsTabUtils.tsx

```diff
@@ -1530,10 +1534,11 @@ export function makeScatterPlotData(
         const sampleCoverageInfo = coverageInformation[d.uniqueSampleKey];
         let profiledMutations = undefined;
         let profiledCna = undefined;
+        const noGenesInAxes = !horzData.hugoGeneSymbol && !vertData.hugoGeneSymbol;
         if (mutations || copyNumberAlterations) {
             const profiledReport = makeScatterPlotData_profiledReport(
-                horzData.hugoGeneSymbol,
-                vertData.hugoGeneSymbol,
+                (noGenesInAxes && selectedGene) ? selectedGene.hugoGeneSymbol : horzData.hugoGeneSymbol,
```

Comment:
this will still return horzData.hugoGeneSymbol even if vertData.hugoGeneSymbol is set but horzData.hugoGeneSymbol is not. is that what you're going for?

Reply:
Yes it is already the case before. The type definition of the first 2 arguments passed to `makeScatterPlotData_profiledReport` is `string|undefined`, so it is okay for one axis to have gene data and the other does not, or both axes does not have any selected gene. I added the condition to account for when gene is selected from the utilities menu above the plot rather than from the axis menu on the side.

Reply:
as you mentioned that selectedGene is only passed through when its a clinical attr vs treatment plot, aka when there are no genes in axes, isnt the `noGenesInAxes` check redundant? cant you just check whether the `selectedGene` parameter is present?